server/controllers/gym/editGym.ts

```diff
@@ -0,0 +1,20 @@
+import { NextFunction, Response, Request } from 'express';
+// import Sequelize from 'sequelize';
+import Gym from '../../database/models/gyms';
+import { CustomError, gymEditSchema } from '../../utils';
+
+export default async function editGymData(req: Request, res: Response, next: NextFunction) {
+  try {
+    await gymEditSchema.validateAsync(req.body);
+    const { id } = req.params;
+    const gym = await Gym.findByPk(id);
+    if (!gym) {
+      throw new CustomError('Gym not found', 404);
+    }
+    await gym.update(req.body);
```

Comment:
What if user changed the logo, then we need to upload image to `aws`,
So check before `gym.logo === req.body.logo`, if those differs then we need to upload image first